app/actions/ModActionPanel/QuickAction.tsx

```diff
@@ -669,9 +669,13 @@ function Form(
                       id="labels"
                       name="labels"
                       formId={FORM_ID}
-                      defaultLabels={currentLabels.filter(
-                        (label) => !isSelfLabel(label),
-                      )}
+                      defaultLabels={currentLabels.filter((label) => {
+                        const serviceDid = client.getServiceDid()
+                        const isExternalLabel = allLabels.find((l) => {
+                          return l.val === label && l.src !== serviceDid
+                        })
```

Comment:
No biggie, but if we're trying to get a bool out (as indicated by the name `isExternalLabel`) then `some()` does that for us.
```suggestion
                        const isExternalLabel = allLabels.some((l) => {
                          return l.val === label && l.src !== serviceDid
                        })
```